test/e2e/apps/deployment.go

```diff
@@ -97,6 +97,11 @@ var _ = SIGDescribe("Deployment", func() {
 	It("test Deployment ReplicaSet orphaning and adoption regarding controllerRef", func() {
 		testDeploymentsControllerRef(f)
 	})
+
+	It("deployment should support proportional scaling", func() {
```

Comment:
I think `It("should support proportional scaling", ...` is better, but the other tests do put `deployment` at the beginning of the sentence.